cluster/config-defaults.yaml

```diff
@@ -490,7 +490,7 @@ tracing_coredns_local_zone_traces_endpoint: ""
 # AMI id given the image name and the Image AWS account owner.
 #
 # [0]: https://github.com/zalando-incubator/cluster-lifecycle-manager/blob/8a9bd1cb2d094038a9e23e646421f8146b48886a/provisioner/template.go#L116
-kuberuntu_image_v1_21: {{ amiID "zalando-ubuntu-kubernetes-production-v1.21.11-master-213" "861068367966"}}
+kuberuntu_image_v1_21: {{ amiID "zalando-ubuntu-kubernetes-production-v1.21.11-master-227" "861068367966"}}
```

Comment:
The newest image is `master-214` (227 is likely the PR number)

Reply:
@rodrigodelmonte fyi: It's going to be `master-216` now (https://sunrise.zalando.net/cdp/ghe/teapot/kubernetes-on-ubuntu?event=)

Reply:
Thanks for the help, I will update it